### DISCUSSION TITLE
Moodle and Gatsby

### DISCUSSION THREAD

**Discussion by 99bits** [gatsbyjs/gatsby#34691] [Help]
Dear fellow Gatsby Comrades,

I'm new to Gatsby and would like to know if it's possible to use Gatsby with Moodle.  
If yes, what would it take to make this kind of integration.  Moodle has a Huge community of users.

Cheers.

**Comment by LekoArts** [comment]
What did you have specifically in mind? I'm not immediately seeing the connection between the two.

**Reply by 99bits** [reply]
I saw integration examples of Gatsby and wordpress and other projects but I didn't see an example integration of Gatsby and Moodle.  Moodle has a large user base that potentially could find Gatsby useful.

**Comment by KyleAMathews** [comment]
As long as Moodle has an API for fetching content — you can build a source plugin to pull that data into Gatsby and build websites with it.

Moodle looks like it has a pretty extensive API — https://docs.moodle.org/dev/Web_service_API_functions

Do people often store a lot of public data in Moodle instances they'd want to build websites from?

**Comment by 99bits** [comment]
Yes, moodle is used by many institutions to deliver online courses that needs to be delivered to many devices.  The use case is exactly the same as for wordpress or other CMS.
